extendeddatacrossword.go

```diff
@@ -25,14 +25,14 @@ func (e *ErrByzantineRow) Error() string {
 	return fmt.Sprintf("byzantine row: %d", e.RowNumber)
 }
 
-// ErrByzantineColumn is thrown when a repaired column does not match the expected column Merkle root.
-type ErrByzantineColumn struct {
-	ColumnNumber uint     // Column index
-	Shares       [][]byte // Pre-repaired column shares. Missing shares are nil.
+// ErrByzantineCol is thrown when a repaired column does not match the expected column Merkle root.
+type ErrByzantineCol struct {
+	ColNumber uint     // Column index
+	Shares    [][]byte // Pre-repaired column shares. Missing shares are nil.
 }
 
-func (e *ErrByzantineColumn) Error() string {
-	return fmt.Sprintf("byzantine column: %d", e.ColumnNumber)
+func (e *ErrByzantineCol) Error() string {
+	return fmt.Sprintf("byzantine column: %d", e.ColNumber)
 }
```

Comment:
I feel like we really should cover all these places in tests, too. But orthogonal to this PR.

Reply:
Agree, opened #58 to track increasing test coverage.